easybuild/easyconfigs/c/CUDAcompat/CUDAcompat-11.6-510.73.08.eb

```
@@ -0,0 +1,43 @@
+name = 'CUDAcompat'
+version = '11.6'
+nv_version = '510.73.08'
+versionsuffix = "-" + nv_version
+
+# Minimal driver requirements for each production branch / major version as found on
+# https://docs.nvidia.com/deploy/cuda-compatibility/index.html#use-the-right-compat-package
+compatible_driver_versions = [
+    '418.40.04',  # (CUDA 10.1)
+    '440.33.00',  # (CUDA 10.2, actually EoL but works)
+    '450.36.06',  # (CUDA 11.0)
```

```suggestion
    '450.36.06',  # (CUDA 11.0)
    '460.73.01',  # (CUDA 11.2)
```
This also works  for 460 drivers (based on my tests)

---

I did additional tests but they are failing for good reason, my driver is a 460 version. Based on the EESSI work I know that `510.85.02` works with the 460 driver.

---

Updated to 510.85.02 and added 460.73.01 (EoL driver) to the compatible list